hooks/common.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import annotations
+
+import argparse
+import logging
+import os
+from collections.abc import Sequence
+
+logger = logging.getLogger(__name__)
+
+
+def setup_logging():
+    logging.basicConfig(
+        level={
+            "error": logging.ERROR,
+            "warn": logging.WARNING,
+            "warning": logging.WARNING,
+            "info": logging.INFO,
+            "debug": logging.DEBUG,
+        }[os.environ.get("PRE_COMMIT_TERRAFORM_LOG_LEVEL", "warning").lower()]
+    )
+
+
+def parse_env_vars(ev_strs: list[str]) -> dict[str, str]:
+    ret = {}
+    for ev_str in ev_strs:
+        name, val = ev_str.split("=", 1)
+        if val.startswith('"') and val.endswith('"'):
+            val = val[1:-1]
+        ret[name] = val
+    return ret
+
+
+def parse_cmdline(
+    argv: Sequence[str] | None = None,
+) -> tuple[list[str], list[str], list[str], list[str], dict[str, str]]:
+    parser = argparse.ArgumentParser(
+        add_help=False,  # to allow us to use -h to be compatible with previous bash version
```

Comment:
The `previous` is a bit confusing. Would you mind of this change suggestion?
```suggestion
        add_help=False,  # Allow the use of `-h` for compatiblity with Bash version of the hook
```

Reply:
done